### DISCUSSION TITLE
Deepgram Response not coming

### DISCUSSION THREAD

**Discussion by pdns90** [deepgram/community#793] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

"error": "cURL error 28: Failed to connect to [api.deepgram.com](http://api.deepgram.com/) port 443: Connection timed out (see https:\/\/[curl.haxx.se](http://curl.haxx.se/)\/libcurl\/c\/libcurl-errors.html) for https:\/\/[api.deepgram.com](http://api.deepgram.com/)\/v1\/listen?model=general"

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
@pdns90, this is an ongoing ISP issue as [we discussed in a prior thread](https://github.com/orgs/deepgram/discussions/764#discussioncomment-9512073).

Currently the only workaround is to use a network or VPN that is not backed by TATA.

Here is another [explanation in our help center](https://deepgram.gitbook.io/help-center/notices/cogent-tata-peering-issue-impact-on-deepgram) that we've added to acknowledge the ongoing issue.
